### ISSUE TITLE
Oocyte maturation arrest

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9886]
**Preferred term label:**
Oocyte maturation arrest

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
620276 
https://pubmed.ncbi.nlm.nih.gov/32666501/
**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
def: Oocyte maturation arrest (OMA) can manifest as failed in vitro fetilization/intracytoplasmic sperm injection (IVF/ICSI) attempts using affected oocytes. 
Comment: Oocyte maturation occurs through a series of molecular and morphological changes. Oocytes are initially arrested at the diplotene stage of prophase I, at which point they are referred to as germinal vesicle (GV) oocytes. Upon a surge in luteinizing hormone, GV oocytes exit prophase I and resume meiosis, and this is followed by chromatin condensation and breakdown of the nuclear envelope. Coupled with spindle formation and chromosome alignment, oocytes enter into meiosis I (when they are referred to as MI oocytes). MI is completed by extrusion of the first polar body. Oocytes then enter into meiosis II (when they are referred to as MII oocytes) but arrest at metaphase of MII until fertilization. In clinical in vitro fertilization (IVF) and intracytoplasmic sperm injection (ICSI), first polar body (PB1) oocytes are regarded as indicative of MII oocytes, which can be used for fertilization. Oocyte maturation arrest occurs at different stages, including the GV stage and MI, leading to female infertility.
PMID:28965849
PMID:37073822

**Comment by pnrobinson** [comment]
added
